server/src/session.ts

```diff
@@ -386,6 +388,25 @@ export class Session {
     return this.tsDefinitionsToLspLocationLinks(definitions);
   }
 
+  private onReferences(params: lsp.TextDocumentPositionParams): lsp.Location[]|undefined {
+    const lsInfo = this.getLSAndScriptInfo(params.textDocument);
+    if (lsInfo === undefined) {
+      return;
+    }
+    const {languageService, scriptInfo} = lsInfo;
+    const offset = lspPositionToTsPosition(scriptInfo, params.position);
+    const references = languageService.getReferencesAtPosition(scriptInfo.fileName, offset);
+    if (!references) {
```

Comment:
prefer explicit check? I don't think we have an official style guide yet, do we? 
In the meantime we should at least be consistent (compare line 393).

Reply:
Oops, yes. I often write these like this as an initial pass because it's less typing >.>